src/frontend/src/hooks/work-packages.hooks.ts

```diff
@@ -86,3 +87,22 @@ export const useDeleteWorkPackage = () => {
     }
   );
 };
+
+/**
+ * Custom React Hook to slack upcoming deadlines.
+ */
+export const useSlackUpcomingDeadlines = (daysUntilDeadline: number) => {
+  const queryClient = useQueryClient();
+  return useMutation<{ message: string }, Error, any>(
+    ['work packages', 'slack upcoming deadlines'],
```

Comment:
don't need to include work packages here or invalidate work packages on success either